apps/builder/app/builder/features/settings-panel/props-section/animation/animation-keyframes.tsx

```diff
@@ -268,11 +275,16 @@ export const Keyframes = ({
                 key={keyRefs.current[index]}
                 value={value}
                 offsetPlaceholder={offsets[index]}
-                onChange={(newValue) => {
+                onChange={(newValue, isEphemeral) => {
```

Comment:
event handler interface is usually better off with a single even object, like. event = {value, isEphemeral}